src/util/EditionCardUtils.tsx

```diff
@@ -267,4 +268,41 @@ export default class EditionCardUtils {
       <>Find in Library Unavailable</>
     );
   }
+
+  static gedEddLink(eddLink: ItemLink, isLoggedIn: boolean) {
+    if (isLoggedIn) {
+      return (
+        <>
+          You can request a partial scan via NYPL&nbsp;
+          <a
+            href="https://www.nypl.org/research/scan-and-deliver"
+            target="_blank"
+            rel="noreferrer"
```

Comment:
why noreferrer?

Reply:
ahh, it seems like my VS linter automatically added that because I have `target="_blank"`... I can use `<Link>` instead so it doesn't complain.

Reply:
yeah I missed that - you should use `Link` instead of `a` for links always, if you're using NYPL design system